### DISCUSSION TITLE
ICU MessageFormat

### DISCUSSION THREAD

**Discussion by igorgregorio** [UnicornsOnLSD/finamp#327] [Translations]
Amazing work! I felt compelled to contribute what little I can to help it reach more people.

I've added translations for Brazilian Portuguese for most strings, but there are 12 that are presenting errors I'm unable to solve. For instance, this string is giving the ICU MessageFormat error, even if the difference between the original string in English and the translated one is of a single character:

Original: {count,plural,=1{{count} item} other{{count} items}
Translated: {count,plural,=1{{count} item} other{{count} itens}

**Comment by UnicornsOnLSD** [comment]
It seems to compile fine, so I've dismissed those warnings. Thanks for translating Finamp!
